pkg/operator/ceph/csi/spec.go

```diff
@@ -370,6 +374,10 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	pluginNodeAffinity := getNodeAffinity(clientset, false)
 	if rbdPlugin != nil {
 		applyToPodSpec(&rbdPlugin.Spec.Template.Spec, pluginNodeAffinity, pluginTolerations)
+		resource := getComputeResource(clientset, false)
+		for _, c := range rbdPlugin.Spec.Template.Spec.Containers {
```

Comment:
We really need the flexibility to apply different requests/limits to the different containers in the pods. The sidecars may have much lower limits, so it wouldn't make sense to apply the same limits.